tools/internal_ci/linux/grpc_python_bazel_test_in_docker.sh

```diff
@@ -27,3 +27,6 @@ python_bazel_tests_single_threaded_unary_streams/bazel_wrapper ${RESULTSTORE_RES
 
 python3 tools/run_tests/python_utils/bazel_report_helper.py --report_path python_bazel_tests_poller_engine
 python_bazel_tests_poller_engine/bazel_wrapper ${RESULTSTORE_RESULTS_FLAG} test --config=python_poller_engine ${BAZEL_FLAGS} ${TEST_TARGETS}
+
+python3 tools/run_tests/python_utils/bazel_report_helper.py --report_path python_bazel_tests_fork_support
+python_bazel_tests_poller_engine/bazel_wrapper ${RESULTSTORE_RESULTS_FLAG} test --config=fork_support --runs_per_test=16 ${BAZEL_FLAGS} //src/python/grpcio_tests/tests/fork:fork_test
```

Comment:
Just curious, is there any conventions for magic numbers like this?

Reply:
I probably should have added a comment here, but this is a number that:

- Is low enough that it won't take forever
- Is high enough that we'll catch _some_ nondetermistic issues if they slip in
- Is low enough that it won't catch the 1/256 issue that I _know_ still exists.